documentation/tutorials/check.md

```diff
@@ -0,0 +1,197 @@
+
+ - `-changes` produces a report of all the changes that will be applied to the schema in the next migration. 
+ - `-drift` produces a report showing objects in the schema which are not the result of any of the currently applied migrations, i.e. changes made outside of Flyway.
+
+More information can be found on [the check command page](/documentation/command/check).
+
+## Install prerequisites
+
+Flyway Check requires .Net 6 so make sure you have this installed. You can download it from [here](https://dotnet.microsoft.com/en-us/download/dotnet/6.0).
+
+## Download the latest Beta version of Flyway Enterprise CLI
+
+First, [download the latest beta version of the Flyway Enterprise CLI](/documentation/learnmore/beta) and extract its contents. 
+
+## Setting up databases
+
+To use `check` we will need access to two databases:
+
+- The **target** database. This is the database we're interested in making changes to. 
+- A **temporary** database. Flyway will clean this database, apply migrations to it, and make comparisons in order to generate **change and drift reports**. 
+
+In this example, there are two databases in our SQL Server instance: `foobar` (the target) and `check_temp_db`. Both can be accessed by the `sa` user.
+
+## Configuration
+
+As well as the usual config parameters (`flyway.url`, `flyway.user`, `flyway.password`, `flyway.licenseKey`...), we also need to configure properties specific to the `check` command (see [the documentation](/documentation/command/check) for more details). In this case, we only need to configure `flyway.check.tempUrl` and `flyway.check.reportFilename` as the other properties all have suitable default values.
+
+Our `flyway.conf` file (found in the `conf` folder), should contain the following values:
+
+```
+flyway.url=jdbc:sqlserver://localhost;databaseName=foobar;trustServerCertificate=true
+flyway.user=sa
+flyway.password=Flyway123
+flyway.licenseKey=<put your license key here>
+
+flyway.check.tempUrl=jdbc:sqlserver://localhost;databaseName=check_temp_db;trustServerCertificate=true
+flyway.check.reportFilename=check_report
+```
+
+## Setting up migrations
+
+In the `sql` folder, we can create a new file `V1__add_table.sql`, containing a simple schema change:
+
+```sql
+create table PERSON (
+    ID int not null,
+    NAME varchar(100) not null
+);
+```
+## Detecting changes using `-changes` flag
+
+Before we run `migrate`, we can use `check` to generate a change report by running it with the `-changes` flag:
+
+<pre class="console">./flyway check -changes</pre>
+
+## Viewing the report output
+
+Having run the command, we can see that Flyway has generated HTML and JSON versions of the `check` report in the working directory. Opening `check-report.html` in a browser, we can clearly see what objects will change in our target schema as a result of running our pending migration(s), expressed in the form of SQL queries. The JSON report contains the same information but in a more machine-readable format. 
```

Comment:
The filename mentioned here is `check-report` but in config it's `check_report` - should make them be the same